doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -304,3 +352,4 @@ flagged for backporting, in which case the status should be changed to
 .. _Integration Testing: ../tests-integration-tests
 .. _Mailing List: ../essentials/#mailing-list
 .. _Submitting Patches: ../essentials/#submitting-patches
+.. _Merging: :ref:`merging`
```

Comment:
No need for this now. You could use the `merging` label as shown here https://github.com/ceph/ceph/commit/ca1cca0abc05d7e3ad4fe25adceb6f83615f66b1

Reply:
@zdover23 could you please take a look at https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref ?